docs/intro.asciidoc

```diff
@@ -48,7 +48,7 @@ Alternatively, you can use a script element and the `elasticApm` global object t
 
 [source,html]
 ----
-<script src="elastic-apm-js-base/dist/bundles/elastic-apm-js-base.umd.min.js"></script>
+<script src="https://unpkg.com/elastic-apm-js-base@2/dist/bundles/elastic-apm-js-base.umd.min.js" crossorigin></script>
```

Comment:
I think we can remove the version from the url, so it always gets the latest version.

Good call to add `crossorigin` 👍

Reply:
yeah makes sense, Lets remove the version and go with latest.